Apps/phcuckoo/cuckoo.json

```diff
@@ -8599,6 +8610,317 @@
+                    "example_values": [
+                        true,
+                        false
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.task_status.target",
+                    "data_type": "string",
+                    "example_values": [
+                        "/tmp/cuckoo-tmp/tmpwwgL_J/eb221521e2eb2651a9318e1487041cca36c75714ee4be548c8bbac160bf0ce03"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.task_status.completed_on",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.data.*.task_status.package",
+                    "data_type": "string",
+                    "example_values": [
+                        "exe"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.task_status.route",
+                    "data_type": "string"
+                },
+                {
+                    "data_path": "action_result.data.*.task_status.timeout",
+                    "data_type": "numeric",
+                    "example_values": [
+                        0
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.task_status.submit_id",
+                    "data_type": "numeric",
+                    "column_name": "submit id",
+                    "column_order": 2,
+                    "example_values": [
+                        58
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.task_status.options.procmemdump",
+                    "data_type": "string",
+                    "example_values": [
+                        "yes"
+                    ]
+                },
+                {
+                    "data_path": "action_result.data.*.task_status.added_on",
+                    "data_type": "string",
+                    "example_values": [
+                        "Sun, 07 Jun 2020 00:18:07 GMT"
+                    ]
+                },
+                {
+                    "data_path": "action_result.status",
+                    "data_type": "string",
+                    "column_name": "status",
+                    "column_order": 0,
+                    "example_values": [
+                        "success"
+                    ]
+                },
+                {
+                    "data_path": "action_result.message",
+                    "data_type": "string",
+                    "example_values": [
+                        "16: eb221521e2eb2651a9318e1487041cca36c75714ee4be548c8bbac160bf0ce03, 17: eb221521e2eb2651a9318e1487041cca36c75714ee4be548c8bbac160bf0ce03"
+                    ]
+                },
+                {
+                    "data_path": "action_result.summary.17",
```

Comment:
How are Summary 17 and 16 decided? What happens if you run the action again? Do you get Summary 18?

Reply:
Fixed